.buildtools/generateStagingSiteInWebpageRepo

```diff
@@ -14,6 +14,6 @@
 # limitations under the License.
 #
 # DEVHINT: RAT-302: Do not use modern JDK due to the javadoc problem; JDK8 and 16 are okay
-./mvnw clean site:site site:stage
+./mvnw clean package site site:stage -DskipTests
```

Comment:
AFAIK tests are needed to calculate the coverage metrics?

Does not site:site trigger an explicit package/install? If the clean is run before, all stuff should be generated freshly.